heroList_GaryMcGinnis.rb

```diff
@@ -0,0 +1,94 @@
+class Heroes
+  @@heroes = []
+  @hero_name = ''
+  @hero_identity = ''
+  @hero_weakness = ''
+
+  def initialize
+      menu
+  end
+
+  def menu
+    choice = 1
+    until choice == 4
+        puts 'What do you want to do?
+              1: Add Hero.
+              2: Read Hero names.
+              3: Delete Hero
+              4: Save File and Quit.'
+        choice = gets.chomp.to_i
+
+        if choice == 1
```

Comment:
This would be nice as a case statement. Although there are a lot of lines of code in each case.